beginner_source/ddp_fault_tolerance.rst

```diff
@@ -0,0 +1,210 @@
+`Introduction <ddp_series_intro.html>`__ \|\| `What is DDP <ddp_theory.html>`__ \|\| `Single-node
+Multi-GPU training <ddp_multigpu.html>`__ \|\| **Fault
+Tolerance** \|\| `Multi-node
+training <../intermediate/ddp_multinode.html>`__ \|\| `mingpt training <../intermediate/ddp_minGPT.html>`__
```

Comment:
In all files: 
Single-node Multi-GPU training => Single-Node Multi-GPU Training
Multi-node training => Multi-Node Training 
mingpt training => minGPT Training